tools/missing-test-detector/testdata/covered_resource_test.go

```diff
@@ -5,10 +5,11 @@ import (
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta/acctest"
 )
 
 func TestAccCoveredResource(t *testing.T) {
-	VcrTest(t, resource.TestCase{
+	acctest.VcrTest(t, resource.TestCase{
```

Comment:
`VcrTest` is also used in other files in the same folder. Should all of `VcrTest` be replaced with `acctest.VcrTest`?

Reply:
Either `VcrTest` or `acctest.VcrTest` will be picked up by the missing test detector after this change.